### PR TITLE
Clarify measurement result-id

### PR DESCRIPTION
**Pull request by laurencelundblade**
Text was a little vague...

This also renames "results-id" to "result-id" as it seems much more correct.

### CODE REVIEW

**Review by quic-jodonogh — APPROVED** [review]

File: draft-ietf-rats-eat.md [hunk 1]
```diff
@@ -972,8 +972,9 @@ For example, it may be desirable to report the results for measurements of the f
 Note that this claim is not for reporting the overall result of a verifier.
 It is solely for reporting the result of comparison to reference values.
 
-An individual measurement result is an array of two, an identifier of the measurement and an enumerated type that is the result.
-The range and values of the measurement identifier varies from one measurement scheme to another.
+An individual measurement result (individual-result) is an array of two, an identifier of the measurement (result-id) and an enumerated type that is the result (result).
```
Comment (nedmsmith) [inline draft-ietf-rats-eat.md:975]:
```suggestion
An individual measurement result (individual-result) is an array consisting of two elements, an identifier of the measurement (result-id) and an enumerated type of the result (result).
```

File: draft-ietf-rats-eat.md [hunk 2]
```diff
@@ -972,8 +972,9 @@ For example, it may be desirable to report the results for measurements of the f
...
+Different measurement systems will measure different things and perhaps measure the same thing in different ways.
```
Comment (nedmsmith) [inline draft-ietf-rats-eat.md:976]:
```suggestion
Different measurement systems may measure different Target Environments, or perhaps measure the same Target Environment differently.
```

File: draft-ietf-rats-eat.md [hunk 3]
```diff
@@ -972,8 +972,9 @@ For example, it may be desirable to report the results for measurements of the f
...
+It is up to each measurement system to define identifiers (result-id) for the measurements it reports.
```
Comment (nedmsmith) [inline draft-ietf-rats-eat.md:977]:
```suggestion
It is up to each Attesting Environment to define the measurement identifiers (result-id) for the measurements it reports.
```

File: draft-ietf-rats-eat.md (hunk 2 above)
Comment (laurencelundblade) [reply draft-ietf-rats-eat.md:976]: While the target of an attestation is a target environment, the target of a measurement system here is not necessary a target environment. This is for use in many other contexts than measured boot or measurements in the TCG sense.

File: draft-ietf-rats-eat.md (hunk 3 above)
Comment (laurencelundblade) [reply draft-ietf-rats-eat.md:977]: Same comment as above. What is a measurement system is very open-ended here.

**Pull request merged**